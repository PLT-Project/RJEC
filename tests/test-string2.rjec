func main () {
    x := "ello";
    for i := 0; i < 4; i = i + 1 {
        printc(x[i]);
    }
    prints(x);

    x[1] = x[2];
    x[3] = x[0];
    for i := 0; i < 4; i = i + 1 {
        printc(x[i]);
    }
    prints(x);
}