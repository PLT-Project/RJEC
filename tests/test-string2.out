e
l
l
o
ello
e
l
l
e
elle